content/en/real_user_monitoring/flutter/mobile_vitals.md

```diff
@@ -18,17 +18,25 @@ further_reading:
 
 Real User Monitoring offers Mobile Vitals, a set of metrics that can help compute insights about your mobile application's responsiveness, stability, and resource consumption. Mobile Vitals range from poor, moderate, to good.
 
-You can view Mobile Vitals for your application by navigating to  **UX Monitoring > Performance Summary** and selecting your application.
+You can see a high level overview of Mobile Vitals for your application by navigating to  **UX Monitoring > Performance Summary** and selecting your application.
 
 {{< img src="real_user_monitoring/flutter/flutter-mobile-vitals-1.png" alt="Mobile Vitals in the Performance Summary tab" style="width:90%;">}}
 
 To access the RUM mobile app performance dashboard, scroll down to the Monitor Performance Metrics section from the application summary page and click **Performance Overview**.
 
 {{< img src="real_user_monitoring/flutter/flutter-performance-overview.png" alt="Mobile performance overview dashboard" style="width:90%;">}}
 
-Understand your application's overall health and performance with the line graphs displaying metrics across various application versions. To filter on application version or see specific sessions and views, click on a graph. 
+Understand your application's overall health and performance with the line graphs displaying metrics across various application versions. To filter on application version or see specific sessions and views, click on a graph.
 
-{{< img src="real_user_monitoring/ios/rum_explorer_mobile_vitals.png" alt="Event Timings and Mobile Vitals in the RUM Explorer" style="width:90%;">}}
+When viewing performance details by the **View** event type, the following event timing and mobile vitals information is available:
+
+- Refresh Rate Average
+- CPU Tickets Per Second
+- Memory Average
+- View Flutter Build TIme Average
```

Comment:
I think this changed to be `Build Time Average` in latest code but @siobhanpmahoney to confirm